lib/api/client-commands/_base-command.ts

```diff
@@ -76,11 +101,14 @@ module.exports = class ClientCommand {
    * @param {function} callback
    * @private
    */
-  executeScriptHandler(method, script, args, callback) {
+
+  transportActions : any = {}
```

Comment:
We should mention all the properties available on `this` at the top of the class, along with `public/private` keywords.